packages/react-native/src/utils/index.ts

```diff
@@ -8,3 +10,13 @@ export const getLineHeight = (fontSize: number): number =>
 export function capitalize<T extends string>(value: T): Capitalize<T> {
   return (value.charAt(0).toUpperCase() + value.slice(1)) as Capitalize<T>;
 }
+
+export function childIsValidComponent<T>(
+  child: ReactChild,
+  componentType: React.FC<T>
```

Comment:
What if the `componentType` param is not a `React.FC`?

Reply:
Removed `childIsValidComponent` function